app/views/projects/_approve_form.html.erb

```diff
@@ -3,8 +3,8 @@
 <div class="container">
   <div class="field">
     <label for="project_directory" class="form-label">Confirm Project Directory</label>
-    <input class="form-directory" type="text" name="project_directory_prefix" required value="<%= @project.project_directory_parent_path %>"></input> /
-    <input class="form-directory-confirm" type="text" name="project_directory" required value=<%= "#{@project.project_directory_short}"%>></input>
+    <input class="form-directory" type="text" name="project_directory_prefix" readonly value="<%= @project.project_directory_parent_path %>"></input> /
```

Comment:
This is where we show the user that all projects go under `/tigerdata` and they have no way of changing it. Notice that we hide the "/princeton" part per Chuck.